core/win32/os.c

```diff
@@ -8597,7 +8597,7 @@ early_inject_init()
     bool under_dr_save;
     where_am_i_t whereami_save;
     wchar_t buf[MAX_PATH];
-    int os_version = get_os_version();
```

Comment:
I cannot see  os_version re-declared in this function?

Reply:
I will move the NULL macro changes to a separate PR.

Reply:
I rolled back the NULL change. There are about 12 warnings of this type when building the complete  dynamorio on Windows.

The reason for os_version into os_version_number is that at os.c at line 112:
static int os_version; 
is declared. This will generate the warning that a local variable hides a global variable declaration.